package.json

```
@@ -1,10 +1,11 @@
 {
-  "name": "react-mdb-template",
-  "version": "1.0.2",
+  "name": "snek-front",
+  "version": "1.0.0",
   "private": true,
+  "homepage": ".",
   "license": "(EUPL-1.2)",
   "dependencies": {
-    "mdbreact": "^4.19.1",
+    "mdbreact": "mdbreact-4.20.0.tgz",
```

This version is outdated. Change it to:
```suggestion
    "mdbreact": "mdbreact-4.26.1.tgz",
```